ui/src/main/java/org/pentaho/di/ui/spoon/trans/LogBrowser.java

```diff
@@ -130,7 +130,7 @@ public void run() {
                 lastNr = Math.min( lastNr, lastLogId.get() + MAX_NR_LOG_LINES_CHUNK );
 
                 List<KettleLoggingEvent> logLines =
-                  KettleLogStore.getLogBufferFromTo( childIds, true, lastLogId.get(), lastNr );
+                  KettleLogStore.getLogBufferFromTo( childIds, false, lastLogId.get(), lastNr );
```

Comment:
@angel-ramoscardona  this change to disable all general log lines in kettle log store. Do you know what is the reason for this change ?